duracloud/duracloud
# Removes RTMP Streaming from all layers of the application.

**Removes RTMP Streaming from all layers of the application.**
* * *

**JIRA Ticket**: https://jira.lyrasis.org/browse/DURACLOUD-1250

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
See above
# How should this be tested?
Currently deployed on Dev instance: 

1. Verify that you can still enable and disable HLS streaming.
2. Verify that there are no traces of RTMP options in the streaming panel in DurAdmin.
3. Verify no references to RTMP streaming in the code base.

# Additional Notes:
TODO remove RTMP from documentation here: https://wiki.lyrasis.org/display/DURACLOUDDOC/DuraCloud+Media+Streaming
# Interested parties
Tag (@ mention) interested parties or, if unsure, @duracloud/committers
